### DISCUSSION TITLE
CloudSim Plus benchmarks, examples and testbeds are missing

### DISCUSSION THREAD

**Discussion by EhsanRasoulpour** [manoelcampos/cloudsim-plus#384] [Q&A] [answered]
Hello
Recently when using the latest version of cloudsimplus, I noticed that when trying to compile it with maven, the following directories does not get created:
/cloudsim-plus-benchmarks/
/cloudsim-plus-examples/
/cloudsim-plus-testbeds/
/cloudsim-plus/

I tried compiling cloudsimplus using IntelliJ IDEA automatic compilation after opening project in it.
I also tried using Apache Maven 3.8.5  and using  "mvn compile" command in the terminal. maybe Im missing something that should also be done. I can build and run examples using the previous version with both intellij and terminal. I could use your help about this problem.

thank you for your time.

**Comment by manoelcampos** [comment, marked as answer]
Such projects are now independent and have their own repository. This repository here just provides the CloudSim Plus API, the only module you need to create your simulations.

Now CloudSim Plus has its own organization here at GitHub, centralizing all CloudSim Plus projects,
including other ones you didn't list above.
Check https://github.com/cloudsimplus to download the projects you want.

**Comment by manoelcampos** [comment]
If you have any questions, feel free to keep in touch.
And please take a second to give CloudSim Plus a star if you haven't yet.
Very appreciated.
